controller.xql

```diff
@@ -22,22 +22,21 @@ if ($exist:path eq '') then
     <dispatch xmlns="http://exist.sourceforge.net/NS/exist">
         <redirect url="{concat(request:get-uri(), '/')}"/>
     </dispatch>
-    
+
 else if ($exist:path = "/") then
     (: forward root path to index.xql :)
     <dispatch xmlns="http://exist.sourceforge.net/NS/exist">
         if(request:get-uri() eq "/exist/apps/dashboard/" and request:get-header("X-Forwarded-URI") eq "/apps/dashboard/") then
             <redirect url="/apps/dashboard/index.html"/>
         else
             <redirect url="index.html"/>
-    }
```

Comment:
Deleting the close curly brace eliminated the syntax error, but to restore intended functionality, the conditional expression inside the `<dispatch>` element need to be encased in curly braces.